aws/resource_aws_wafv2_regex_pattern_set_test.go

```diff
@@ -2,6 +2,7 @@ package aws
 
 import (
 	"fmt"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/tfawsresource"
```

Comment:
Nitpick: stdlib imports usually at top separated by space and then third party imports
```
import (
    "fmt"
    
    "other/imports"
)
```

Reply:
all set!